### DISCUSSION TITLE
Content duplication on Netlify deploy while it is fine locally

### DISCUSSION THREAD

**Discussion by JeremieLitzler** [vuepress-theme-hope/vuepress-theme-hope#2484] [Q&A] [answered]
Hello,

I'm using a Node 16.13 and yarn 1.22.11.
My website used vuepress 2 beta 53 with Mr Hope awesome theme in beta 132.

I have this [web page](https://iamjeremie.me/posts/2022-12-02-these-8-free-ingredients-will-bulk-out-your-compost-bins-huw-richards/) (for example, but I see the same behavior on all post pages) deployed by a netlify build from [this repossitory](https://github.com/JeremieLitzler/journal) on the main branch.

I see this on the production URL:
![image](https://user-images.githubusercontent.com/2410556/205564576-d98421e5-cb2d-418a-a4f9-8d0e7d833b30.png)

Locally, I see that, which is the expected result:
![image](https://user-images.githubusercontent.com/2410556/205564518-41a92ebb-6596-455b-8b1d-adc36604cdf6.png)

When I load the production page, I see the right content for a second and then it seems to be replaced:
![These 8 Free Ingredients Will Bulk Out Your Compost Bins, by Huw Richards _ A shared journal by Jeremie Litzler](https://user-images.githubusercontent.com/2410556/205565388-55a2f144-b369-48bd-a3fa-c8f0e22e4deb.gif)

Somehow, I also find some of the content repeated twice as I scroll... I am not sure why this happens.

I don't use the Asset optimization nor the Prerendering option (there is no difference with my issue anyways).

What could be the source of my issue?

Thanks.

**Comment by JeremieLitzler** [comment]
OK, and if I deploy the same repo and branch to another instance, the page looks fine if I access it from the home page > `Read` button and select the article in the list...

If I do `CTRL + SHIFT + F5` on the article page, the issue occurs and I can't get out of it unless I start back from the homepage...

**Comment by Mister-Hope** [comment, marked as answer]
Hi, I spent 10 minutes but I failed to findout any problems, you are using standard html tags, and the console is not showing you got a hygration mismatch,

Also other pages seems not having these kind of issue.

You may try to make a mininal repo to see if this is related to your markdown content by trying to create a new template and add your notes inside it. If you can reproduce it, welcome to try editing that file to see what part triggers this issue, and show that repo link and I may investigate further.

If you can not reproduce it stablely, then I am afraid I have no ideas about how this issue happens, it's annoying to debug these kind of questions in production without any warning or errors.

**Reply by JeremieLitzler** [reply]
Hi @Mister-Hope 

I have found the source of the problem.

When I have the image title, the image disappears:

```md
![Someone cutting vegetables in a garden](./images/2022-12-05-the-art-of-lazy-composting-how-to-make-high-quality-compost-the-simple-way-huw-richards.jpg 'Often the compost comes from the waste of the vegetable garden. Credits: image taken from Huw Richards’s vlog')
```

When I don't use it, it is displayed fine:

```md
![Someone cutting vegetables in a garden](./images/2022-12-05-the-art-of-lazy-composting-how-to-make-high-quality-compost-the-simple-way-huw-richards.jpg)
```

See the example on [this page](https://fix-missing-and-duplicated-content.netlify.app/posts/2022-12-06-the-art-of-lazy-composting-how-to-make-high-quality-compost-the-simple-way-huw-richards/) from [this repo branch](https://github.com/JeremieLitzler/journal/tree/fix-production-content-duplication/docs/posts/2022-12-06-the-art-of-lazy-composting-how-to-make-high-quality-compost-the-simple-way-huw-richards).

It seems that there is a bug in the image title enhancement option only in production while in development, the issue is not there.

BTW, you will see the same bug on your page: https://vuepress-theme-hope.github.io/v2/guide/markdown/image.html#image-size
![image](https://user-images.githubusercontent.com/2410556/205842307-fbb3ffa7-e956-4a0d-92c0-6976d3f1197b.png)

Let me know if you need more information on the matter.

**Reply by JeremieLitzler** [reply]
Hi @Mister-Hope 

I will try your fix from https://github.com/vuepress-theme-hope/vuepress-theme-hope/releases/tag/v2.0.0-beta.134 tomorrow!

Thanks for the quick turnaround :D

**Reply by JeremieLitzler** [reply]
@Mister-Hope 

beta 134 works fine for me. Thanks.
